chainer/function.py

```diff
@@ -7,26 +7,23 @@
 import chainer
 from chainer import configuration
 from chainer import cuda
-from chainer import flag
 from chainer.utils import type_check
 from chainer import variable
 
 
 def no_backprop_mode():
-    """Disable back-propagation for Variable whose volatile is auto.
+    """Disable back-propagation.
```

Comment:
How about adding the word "context manager" to the description of this function. For example, "Returns a context manager that disables back-propagation".